realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -684,10 +684,16 @@ private Scanner getFullStringScanner(InputStream in) {
 
     /**
      * Instantiates and adds a new object to the Realm.
+     * <p>
+     * This method is only available for the model classes that have no primary key defined
+     * even though the class provides a default value for it.
```

Comment:
This method is only available for model classes with no `@PrimaryKey` annotation. If you like to create an object that has a primary key, use `createObject(Class, Object)` or `copyToRealm(RealmModel)` instead.